faker.go

```diff
@@ -158,20 +150,18 @@ func (f Faker) UInt64() uint64 {
 // IntBetween returns a fake Int between a given minimum and maximum values for Faker
 func (f Faker) IntBetween(min, max int) int {
 	diff := max - min
-
-	if diff < 0 {
-		diff = 0
-	}
-
+	var rndInt int
```

Comment:
nitpick: can you rename the `rndInt` to just `value`, following the standard of the rest of the methods.